README.md

```diff
@@ -114,9 +114,9 @@ If you would like to use or build the mlpack Python bindings, make sure that the
 following Python packages are installed:
 
       setuptools
-      cython >= 0.24
-      numpy
-      pandas >= 0.15.0
+      cython >= 0.23.4
+      numpy  >= 1.11.0
+      pandas >= 0.17.1
```

Comment:
Why did you change the minimum package versions here?  These are what's required to build the Python bindings, so, I think they were fine as-is, unless you verified that everything works with Cython 0.23.4.  Let me know what you think or if I overlooked something. :+1:

Reply:
I was trying to install Cython 0.23.4 but it is supported for only python versions below 3.6 and getting one to work was very difficult because there were python conflicts even with python3.5.x being installed and as nothing below Cython 0.26.1 is available from Ubuntu 18.04, installing from deb packages was giving conflicts so i will return the minimum package versions as they were and except that i have check the package names are all valid in this PR with Ubuntu(apt) and fedora(dnf).